src/features/ProductCategoryPage.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { useGetProductsByCategoryQuery } from "./api/apiSlice";
+
+export default function ProductCategoryPage() {
+  const { category } = useParams();
```

Comment:
maybe category as page title? and loading state only switches the product list?

Reply:
Done.